Composer/packages/client/src/pages/design/exportSkillModal/content/SelectProfile.tsx

```diff
@@ -173,38 +244,49 @@ export const SelectProfile: React.FC<ContentProps> = ({
     }
   }, [id]);
 
-  return isProfileValid ? (
+  const handleShowCreateProfileDialog = () => {
+    setShowCreateProfileDialog(true);
+  };
+
+  return !showCreateProfileDialog ? (
     <div css={styles.container}>
       <Dropdown
         required
         ariaLabel={formatMessage('Select a publishing profile')}
         label={formatMessage('Publishing Profile')}
         options={publishingOptions}
         placeholder={formatMessage('Select one')}
-        selectedKey={publishingOptions[0].key}
+        selectedKey={selectedKey}
         styles={{ root: { paddingBottom: '8px' } }}
         onChange={handleCurrentProfileChange}
+        onRenderTitle={onRenderTitle}
       />
-      <TextField
-        disabled
-        required
-        ariaLabel={formatMessage('The endpoint url')}
-        label={formatMessage('Endpoint Url')}
-        placeholder={formatMessage('The endpoint url of your web app resource')}
-        styles={{ root: { paddingBottom: '8px' } }}
-        value={endpointUrl}
-        onRenderLabel={onRenderLabel}
-      />
-      <TextField
-        disabled
-        required
-        ariaLabel={formatMessage('The app id of your application registration')}
-        label={formatMessage('Microsoft App Id')}
-        placeholder={formatMessage('The app id')}
-        styles={{ root: { paddingBottom: '8px' } }}
-        value={appId}
-        onRenderLabel={onRenderLabel}
-      />
+      {!isValidProfile(currentTarget) ? (
+        onRenderInvalidProfileWarning(hasValidProfile, handleShowCreateProfileDialog)
+      ) : (
+        <Fragment>
+          <TextField
+            disabled
+            required
+            ariaLabel={formatMessage('The endpoint url')}
+            label={formatMessage('Endpoint Url')}
+            placeholder={formatMessage('The endpoint url of your web app resource')}
+            styles={{ root: { paddingBottom: '8px' } }}
+            value={endpointUrl}
+            onRenderLabel={onRenderLabel}
+          />
+          <TextField
+            disabled
+            required
+            ariaLabel={formatMessage('The app id of your application registration')}
+            label={formatMessage('Microsoft App Id')}
+            placeholder={formatMessage('The app id')}
```

Comment:
can you consult UX for a better placeholder?

Reply:
@vivekkshankar  Could u give me a better placeholder? thanks